### ISSUE TITLE
Null pointer when new checker isn't in a package

### ISSUE THREAD

**Issue by csgordon** [typetools/checker-framework#845]
If someone tries to write a new checker outside the checker framework, and doesn't explicitly put the checker into a package, the checker framework gets a null pointer exception here: https://github.com/typetools/checker-framework/blob/checker-framework-2.1.0/framework/src/org/checkerframework/framework/type/AnnotationClassLoader.java#L110

Probably ```getPackage()``` returns null.

Instead we should issue an error message.

**Comment by smillst** [comment]
Thanks for submitting the issue.  We think it's ok for a checker not to be in a package and will fix the null pointer exception.

**Comment by smillst** [comment]
Closed via eedf7d6deb8f2ebbd0ec7e504c3cea8ebba07a55
